src/content/updates/migration/designers.mdx

```diff
@@ -0,0 +1,158 @@
+---
```

Comment:
Designer Job to be done: set up my design environment to use v10 components, resources, and styles to create new design artifacts or update existing design artifacts. 
Questions: can v9 sketch libraries and v10 libraries co-exist in my sketch set up? will my documents automatically change if I _replace_ v9 libraries with v10 libraries?

Reply:
@chrisconnors-ibm I'm not totally sure how that _replacing_ action would happen in Sketch. Is there a way to do that currently? Maybe I'm misunderstanding the question? 

I would say that it is technically possible for v9 and v10 libraries to co-exist in Sketch, but that mixing v9 and v10 components in the same layout is not an approved use of the design system. I'll add this language.

Reply:
>I'm not totally sure how that replacing action would happen in Sketch. 
it won't between v 9 and 10, but there is clearly some expectation that it will or could.

>I'm not totally sure how that replacing action would happen in Sketch. Is there a way to do that currently? Maybe I'm misunderstanding the question?
This is what Sketch does when they update your libraries. We will do this until the next Design Language change if possible (ie v10 to v11 will be a library update, not a replacement)

>I would say that it is technically possible for v9 and v10 libraries to co-exist in Sketch, but that mixing v9 and v10 components in the same layout is not an approved use of the design system. 

They can definitely co-exist in people's sketch environments which is a good thing since for some time people will probably need to work on both systems.

Reply:
We have this language already. @chrisconnors-ibm Do you think this sufficiently covers this user story? _Please note that upgrading to the v10 kit will not automatically change any files created in previous versions of the kit. In Sketch, the migration of existing v9 layouts will be a manual process for the designer. Also note that even though libraries from multiple versions of Carbon can exist at the same time in Sketch, using v10 components in the same layout as pre-v10 components is not an approved use of the design system._